<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnuccountingFireForm</class>
 <widget class="QDialog" name="UnuccountingFireForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add unuccounting fire departure</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_18">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lbDepType">
           <property name="text">
            <string>Departure type</string>
           </property>
           <property name="buddy">
            <cstring>lineEditDepType</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditDepType">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>неучетный пожар</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lbTypeUnuccountingFire">
           <property name="text">
            <string>Type unuccounting fire</string>
           </property>
           <property name="buddy">
            <cstring>cBoxTypeUnuccountingFire</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="cBoxTypeUnuccountingFire"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="lbAddress">
           <property name="text">
            <string>Address</string>
           </property>
           <property name="buddy">
            <cstring>lineEditAddress</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditAddress">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="lbReported">
           <property name="text">
            <string>Reported</string>
           </property>
           <property name="buddy">
            <cstring>lineEditReported</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditReported">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="lbDepDate">
           <property name="text">
            <string>Departure date]</string>
           </property>
           <property name="buddy">
            <cstring>dateEditDeparture</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEditDeparture"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="lbDepTime">
           <property name="text">
            <string>Departure time</string>
           </property>
           <property name="buddy">
            <cstring>timeEditDeparture</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QTimeEdit" name="timeEditDeparture"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="lbArrivalDate">
           <property name="text">
            <string>Arrival date</string>
           </property>
           <property name="buddy">
            <cstring>dateEditArrival</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEditArrival"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="lbArrivalTime">
           <property name="text">
            <string>Arrival time</string>
           </property>
           <property name="buddy">
            <cstring>timeEditArrival</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QTimeEdit" name="timeEditArrival"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="lbFirstBarrelDate">
           <property name="text">
            <string>First barrel date</string>
           </property>
           <property name="buddy">
            <cstring>dateEditFirstBarrel</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEditFirstBarrel"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="lbFirstBarrelTime">
           <property name="text">
            <string>First barrel time</string>
           </property>
           <property name="buddy">
            <cstring>timeEditFirstBarrel</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QTimeEdit" name="timeEditFirstBarrel"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="lbLocDate">
           <property name="text">
            <string>Localization date</string>
           </property>
           <property name="buddy">
            <cstring>dateEditLocalization</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_11">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEditLocalization"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="lbLocalizationTime">
           <property name="text">
            <string>Localization time</string>
           </property>
           <property name="buddy">
            <cstring>timeEditLocalization</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_12">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QTimeEdit" name="timeEditLocalization"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="lbMitigationDate">
           <property name="text">
            <string>Mitigation date</string>
           </property>
           <property name="buddy">
            <cstring>dateEditMitigation</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_13">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEditMitigation"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="lbMitigationTime">
           <property name="text">
            <string>Mitigation time</string>
           </property>
           <property name="buddy">
            <cstring>timeEditMitigation</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_14">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QTimeEdit" name="timeEditMitigation"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QLabel" name="lbReturnDate">
           <property name="text">
            <string>Return date</string>
           </property>
           <property name="buddy">
            <cstring>dateEditReturn</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_15">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEditReturn"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <widget class="QLabel" name="lbReturnTime">
           <property name="text">
            <string>Return time</string>
           </property>
           <property name="buddy">
            <cstring>timeEditReturn</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_16">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QTimeEdit" name="timeEditReturn"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="lbDescription">
           <property name="text">
            <string>Description</string>
           </property>
           <property name="buddy">
            <cstring>pTextEditdescription</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="pTextEditdescription">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_19">
     <item>
      <widget class="QPushButton" name="pbApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_17">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>UnuccountingFireForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>603</x>
     <y>522</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbApply</sender>
   <signal>clicked()</signal>
   <receiver>UnuccountingFireForm</receiver>
   <slot>addUnuccountingFireDeparture()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>522</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateEditDeparture</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>dateEditArrival</receiver>
   <slot>setDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeEditDeparture</sender>
   <signal>timeChanged(QTime)</signal>
   <receiver>timeEditArrival</receiver>
   <slot>setTime(QTime)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateEditArrival</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>dateEditFirstBarrel</receiver>
   <slot>setDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateEditFirstBarrel</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>dateEditLocalization</receiver>
   <slot>setDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateEditLocalization</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>dateEditMitigation</receiver>
   <slot>setDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateEditMitigation</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>dateEditReturn</receiver>
   <slot>setDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeEditArrival</sender>
   <signal>timeChanged(QTime)</signal>
   <receiver>timeEditFirstBarrel</receiver>
   <slot>setTime(QTime)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeEditFirstBarrel</sender>
   <signal>timeChanged(QTime)</signal>
   <receiver>timeEditLocalization</receiver>
   <slot>setTime(QTime)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeEditLocalization</sender>
   <signal>timeChanged(QTime)</signal>
   <receiver>timeEditMitigation</receiver>
   <slot>setTime(QTime)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeEditMitigation</sender>
   <signal>timeChanged(QTime)</signal>
   <receiver>timeEditReturn</receiver>
   <slot>setTime(QTime)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>484</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditAddress</sender>
   <signal>editingFinished()</signal>
   <receiver>UnuccountingFireForm</receiver>
   <slot>resizeLineEditAddress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addUnuccountingFireDeparture()</slot>
  <slot>resizeLineEditAddress()</slot>
 </slots>
</ui>
